Subject: Revert “Bug 601926 - Don't hardcode meta to alt”

This reverts an upstream patch in 0.25.90 that broke the Alt modifier
key.

Origin: reverse of http://git.gnome.org/browse/vte/commit/?id=b73782a28894e25ed146271f9d6c6775a6836199
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/vte/+bug/619754
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=601926
Last-Update: 2009-08-18

diff --git a/src/keymap.h b/src/keymap.h
index 243e22e..3a4cefe 100644
--- a/src/keymap.h
+++ b/src/keymap.h
@@ -27,7 +27,7 @@
 
 G_BEGIN_DECLS
 
-#define VTE_META_MASK		GDK_META_MASK
+#define VTE_META_MASK		GDK_MOD1_MASK
 #define VTE_NUMLOCK_MASK	GDK_MOD2_MASK
 
 /* Map the specified keyval/modifier setup, dependent on the mode, to either
